geode-assembly/src/acceptanceTest/java/org/apache/geode/launchers/ServerStartupRedundancyRecoveryNotificationTest.java

```diff
@@ -65,14 +65,22 @@ public void redundantRegionThatRequiresRedundancyRecovery() throws IOException {
     Files.createDirectories(server1Folder);
     server2Folder = temporaryFolder.newFolder(SERVER_2_NAME).toPath().toAbsolutePath();
 
-    locatorPort = AvailablePortHelper.getRandomAvailableTCPPort();
+    int[] ports = AvailablePortHelper.getRandomAvailableTCPPorts(3);
+
+    int senderLocatorJmxPort = ports[0];
+    int senderLocatorHttpPort = ports[1];
+    locatorPort = ports[2];
+
 
     String startLocatorCommand = String.join(" ",
         "start locator",
         "--name=" + LOCATOR_NAME,
         "--dir=" + locatorFolder,
         "--port=" + locatorPort,
-        "--locators=localhost[" + locatorPort + "]");
+        "--locators=localhost[" + locatorPort + "]",
+        "--J=-Dgemfire.jmx-manager-start=true",
+        "--J=-Dgemfire.jmx-manager-http-port=" + senderLocatorHttpPort,
+        "--J=-Dgemfire.jmx-manager-port=" + senderLocatorJmxPort);
```

Comment:
This isn't necessary. (It runs fine without these extra properties.)

Reply:
Fixed